docs/src/user-guide/configuring/plugins.md

```diff
@@ -1,93 +1,26 @@
 ---
-title: Plugins
+title: Plugins & Parsers
+layout: doc
```

Comment:
```suggestion
title: Plugins & Parsers
```

`layout: doc` is the default now (https://github.com/eslint/eslint/pull/16484), so it isn't necessary here.